contracts/swap/swap.go

```diff
@@ -91,6 +93,12 @@ func InstanceAt(address common.Address, backend chain.Backend) (Contract, error)
 	return c, err
 }
 
+// Bounced obtains if a cheque has been bounced previously of a contract at a specific address.
```

Comment:
i find the term "a specific address" kind of confusing, i started to check if it was maybe a parameter.

as i understand it, this returns the boolean indicating if a cheques has been bounced when attempting to cash it in the receiver contract, no?

if so, please update this comment to accurately reflect this